src/components/Seo.tsx

```diff
@@ -7,8 +7,14 @@
 
 import React from 'react'
 import Helmet from 'react-helmet'
-import { useStaticQuery, graphql } from 'gatsby'
+import { useStaticQuery, graphql, withAssetPrefix } from 'gatsby'
+// import WebFont from 'webfontloader'
```

Comment:
should this be uncommented?

Reply:
ah, nvm, i see `OpenSans.css` further down in the PR – maybe this should just be removed entirely?

Reply:
Yes done!